coderd/userauth.go

```diff
@@ -333,7 +349,7 @@ func (api *API) loginRequest(ctx context.Context, rw http.ResponseWriter, req co
 	// If the user logged into a suspended account, reject the login request.
 	if roles.Status != database.UserStatusActive {
 		httpapi.Write(ctx, rw, http.StatusUnauthorized, codersdk.Response{
-			Message: "Your account is suspended. Contact an admin to reactivate your account.",
+			Message: fmt.Sprintf("Your account is %s. Contact an admin to reactivate your account.", roles.Status),
```

Comment:
If for some reason, even due to a bug, Coder will show this message, it will be accurate - "Your account is dormant" or "Your account is suspended". Otherwise, the message about suspended account will be present also for dormant accounts.

Reply:
That's fair 👍 nice future-proofing.